src/js/filter.js

```diff
@@ -1,43 +1,47 @@
 /* global document, window */
 
-const setUpFilter = (markerGroup, citiesToMarkers, data) => {
-  const filter = document.querySelector(".filter--scope");
-  const priorSelection = [
-    "Regional",
-    "Citywide",
-    "City Center",
-    "Transit Oriented",
-    "Main Street",
-  ];
```

Comment:
I removed this to instead simply set `true` to every option in the list. That's simpler. Yes it's less general for possible future changes. But better is to keep things as simple as possible. Avoid "premature generalization"

For example, there was a bug in the previous implementation! This wasn't updated with the new values for some of the options.